packages/kit/src/exports/vite/build/build_service_worker.js

```diff
@@ -69,7 +69,7 @@ export async function build_service_worker(
 			lib: {
 				entry: /** @type {string} */ (service_worker_entry_file),
 				name: 'app',
-				formats: ['es']
+				formats: ['iife']
 			},
 			rollupOptions: {
```

Comment:
```suggestion
				formats: ['es']
			},
			// avoid https://github.com/vitejs/vite/issues/15379
			minify: 'terser',
			rollupOptions: {
```

Reply:
Nevermind. That would require terser to be installed. Your workaround was better